sdk/src/dataMapper/ModuleDataMapper.ts

```diff
@@ -41,7 +41,9 @@ export default class ModuleDataMapper extends BaseDataMapper<Module, Module_filt
     validationPayloads: string[],
     value: number,
   ) {
-    const matchingSchema = await this.veraxSdk.schema.findOneById(attestationPayload.schemaId);
+    const matchingSchema = this.findOneSchemaById
+      ? await this.findOneSchemaById(attestationPayload.schemaId)
+      : undefined;
```

Comment:
More percise, `!matchingSchema` return true can now be either "No matching Schema" or the given `findOneSchemaById` is undefined, and if that's the case, I think the msg in`throw new Error("No matching Schema")` need to change accordingly